app/assets/stylesheets/spree/frontend/solidus_paypal_braintree.css

```diff
@@ -16,14 +16,24 @@ the installer will append this file to the app vendored assets here: 'vendor/ass
 }
 
 .apple-pay-button {
-  display: none;
-  background-color: black;
-  background-image: -webkit-named-image(apple-pay-logo-white);
-  background-size: 100% 100%;
-  background-origin: content-box;
+  -webkit-appearance: -apple-pay-button;
+  -apple-pay-button-type: buy;
+  visibility: hidden;
+  display: inline-block;
+  min-height: 30px;
+  border: 1px solid black;
+  background-image: -webkit-named-image(apple-pay-logo-black);
+  background-size: 100% calc(60% + 2px);
   background-repeat: no-repeat;
-  width: 100%;
-  height: 44px;
-  padding: 10px 0;
-  border-radius: 10px;
+  background-color: white;
+  background-position: 50% 50%;
+  border-radius: 5px;
+  padding: 0;
+  margin: 5px auto;
+  transition: background-color .15s;
+  width: auto;
+}
+
+.apple-pay-button.visible {
+  visibility: visible;
```

Comment:
I think we should use `display` over `visibility` as the latter still occupies space, while the former not.
We don't want to have holes in the layout if the payment va Apple Pay is not possible. But this can also be tackled in a later PR and should not hold this already very large PR back.